resources/assets/js/components/Application/Experience/Experience.tsx

```diff
@@ -0,0 +1,742 @@
+            handleDelete={handleDelete}
+            handleEdit={handleEdit}
+            institution={education.institution}
+            irrelevantSkillCount={irrelevantSkillCount}
+            isActive={education.is_active}
+            isEducationJustification={education.is_education_requirement}
+            relevantSkills={relevantSkills}
+            showButtons
+            showSkillDetails
+            startDate={education.start_date}
+            status={educationStatus}
+            thesisTitle={education.thesis_title}
+          />
+        );
+      case "experience_work":
+        return (
+          <ExperienceWorkAccordion
+            key={`${work.id}-${work.type}`}
+            endDate={work.end_date}
+            group={work.group}
+            handleDelete={handleDelete}
+            handleEdit={handleEdit}
+            irrelevantSkillCount={irrelevantSkillCount}
+            isActive={work.is_active}
+            isEducationJustification={work.is_education_requirement}
+            organization={work.organization}
+            relevantSkills={relevantSkills}
+            showButtons
+            showSkillDetails
+            startDate={work.start_date}
+            title={work.title}
+          />
+        );
+      case "experience_community":
+        return (
+          <ExperienceCommunityAccordion
+            key={`${community.id}-${community.type}`}
+            endDate={community.end_date}
+            group={community.group}
+            handleDelete={handleDelete}
+            handleEdit={handleEdit}
+            irrelevantSkillCount={irrelevantSkillCount}
+            isActive={community.is_active}
+            isEducationJustification={community.is_education_requirement}
+            project={community.project}
+            relevantSkills={relevantSkills}
+            showButtons
+            showSkillDetails
+            startDate={community.start_date}
+            title={community.title}
+          />
+        );
+      case "experience_personal":
+        return (
+          <ExperiencePersonalAccordion
+            key={`${personal.id}-${personal.type}`}
+            description={personal.description}
+            endDate={personal.end_date}
+            handleDelete={handleDelete}
+            handleEdit={handleEdit}
+            irrelevantSkillCount={irrelevantSkillCount}
+            isActive={personal.is_active}
+            isEducationJustification={personal.is_education_requirement}
+            isShareable={personal.is_shareable}
+            relevantSkills={relevantSkills}
+            showButtons
+            showSkillDetails
+            startDate={personal.start_date}
+            title={personal.title}
+          />
+        );
+      case "experience_award":
+        return (
+          <ExperienceAwardAccordion
+            key={`${award.id}-${award.type}`}
+            // TODO: Add awardLink field to Award Modal Form?
+            awardLink={{ text: "My Award", title: "my award", url: "/" }}
```

Comment:
I think I had removed this field from the Award Accordion in the Review PR, seems to have been added as an oversight to the markup.

I can also fix this up in my Review PR if you'd like.

Reply:
Again, sounds good 👍